pallets/task/src/tests.rs

```diff
@@ -1,23 +1,46 @@
 use crate::{mock::*, Error};
-use frame_support::{assert_noop, assert_ok};
+use frame_support::{assert_noop, assert_ok, traits::{UnixTime, Hooks}};
 use pallet_balances::Error as BalancesError;
 
-pub const DEADLINE:u32 = 77;
 pub const USERNAME:[u8; 1] = [7];
-pub const TITLE:[u8; 1] = [1];  
+pub const TITLE:[u8; 1] = [1];
+pub const MILLISEC_PER_BLOCK: u64 = 12000; // 12 sec for a block
```

Comment:
It's better to assume a shorter block time.. something like 6 seconds is better.